pkg/resources/database_acceptance_test.go

```diff
@@ -83,6 +89,50 @@ func TestAcc_Database(t *testing.T) {
 	})
 }
 
+func TestAcc_DatabaseRemovedOutsideOfTerraform(t *testing.T) {
+	id := generateUnsafeExecuteTestDatabaseName(t)
+
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: acc.TestAccProtoV6ProviderFactories,
+		PreCheck:                 func() { acc.TestAccPreCheck(t) },
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			tfversion.RequireAbove(tfversion.Version1_5_0),
+		},
+		CheckDestroy: testAccCheckDatabaseExistence(t, id, false),
```

Comment:
should this function be moved into the database acceptance tests? it's strange that this function is in a different file. because i imagine we will be having more of these check destroy functions in the future, and these functions should be in the file for the object. or maybe we make one generic check destroy function that can handle any type.

Reply:
instead of putting this in the checkdestroy, when should probably put this in a custom check function. there is actually an example here of this exact use case https://developer.hashicorp.com/terraform/plugin/sdkv2/testing/acceptance-tests/teststep#custom-check-functions

Reply:
I can move it to helper but this will result in new pumped helper file.

I want to have it in check destroy, because I check after destroy, not a custom test check which is done after step.